src/EPR.Calculator.API.Data/DataModels/CalculatorRunOrganisationDataDetail.cs

```diff
@@ -1,24 +1,25 @@
-﻿using Microsoft.EntityFrameworkCore;
-using System.ComponentModel.DataAnnotations;
+﻿using System.ComponentModel.DataAnnotations;
 using System.ComponentModel.DataAnnotations.Schema;
 using System.Diagnostics.CodeAnalysis;
 
 namespace EPR.Calculator.API.Data.DataModels
 {
     [ExcludeFromCodeCoverage]
     [Table("calculator_run_organization_data_detail")]
-    [PrimaryKey(nameof(OrganisationId), nameof(SubsidaryId))]
 
     public class CalculatorRunOrganisationDataDetail
     {
+        [Required]
+        public int Id { get; set; }
+
         [StringLength(400)]
         [Column("organisation_id")]
 
-        public required string OrganisationId { get; set; }
+        public string? OrganisationId { get; set; }
 
         [StringLength(400)]
         [Column("subsidiary_id")]
-        public required string SubsidaryId { get; set; }
+        public string? SubsidaryId { get; set; }
 
```

Comment:
I can't see the submission_period_desc column which is in the ADR.

Reply:
It will be part of a different ticket.

Reply:
Agreed  with @johnbenjamin-defra in a call that this will be a different ticket